Branch-SDK/src/io/branch/referral/SystemObserver.java

```diff
@@ -322,6 +323,33 @@ public String getPhoneModel() {
     }
 
     /**
+     * Gets default ISO2 Country code
+     *
+     * @return A string representing the ISO2 Country code (eg US, IN)
+     */
+    public String getISO2CountryCode() {
+        if (Locale.getDefault() != null) {
+            return Locale.getDefault().getCountry();
+        } else {
+            return "";
```

Comment:
Is there a case where these return an empty string? Or is this just to be safe. I don't think we should get in the habit of sending empty strings in the POST body.